webdriver-spec.html

```diff
@@ -2547,6 +2547,66 @@ <h3>Processing Capabilities</h3>
 </ol>
 
 </section> <!-- /Processing Capabilities -->
+
+<aside class=example>
+  <p>Capabilities are meant for defining a failsafe matrix selection of a
+    browser configuration in a distributed WebDriver environment. When
+    communicating directly with an endpoint node (i.e. geckodriver),
+    <a>capabilities</a> dictionaries are only used for carrying
+    browser-specific configuration (i.e. <var>moz:firefoxOptions</var>);
+    however, when talking to an <a>intermediary node</a> multiplexer,
+    the request will likely be forwarded to another node: <a>capabilities</a>
+    are used for selecting a node in this network that matches the capability
+    requirements.</p>
+
+    <p>If there is a requirement to have a Firefox webdriver, with no specific
+      requirements in terms of operating system, the requirement object
+      would be:</p>
+
+<pre>{
```

Comment:
This example is incorrect. Provided the sent payload contains an object as the value of the `capabilities` property, the spec says it's fine to do what you'd like. In other words, a remote end can do what it likes when it sees an empty capabilities payload.